command/sub8_mission_control/action/current_goal.action

```diff
@@ -0,0 +1,8 @@
+#goal definition
+nav_msgs/Odometry order
+---
+#result definition
+bool flag
+---
+#feedback
+int32[] sequence
```

Comment:
What does the list of integers contain, though? How does the client or server know how to decode it?